src/main/java/uk/gov/justice/hmpps/probationteams/controllers/ProbationAreaController.kt

```diff
@@ -0,0 +1,192 @@
+package uk.gov.justice.hmpps.probationteams.controllers
+
+import io.swagger.annotations.*
+import  lombok.extern.slf4j.Slf4j
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import uk.gov.justice.hmpps.probationteams.dto.ErrorResponse
+import uk.gov.justice.hmpps.probationteams.dto.LocalDeliveryUnit2Dto
+import uk.gov.justice.hmpps.probationteams.dto.ProbationTeamDto
+import uk.gov.justice.hmpps.probationteams.model.LocalDeliveryUnit2
+import uk.gov.justice.hmpps.probationteams.services.DeleteOutcome
+import uk.gov.justice.hmpps.probationteams.services.LocalDeliveryUnit2Service
+import uk.gov.justice.hmpps.probationteams.services.SetOutcome
+
+@Api(tags = ["probation-areas"])
+@RestController
+@RequestMapping(
+        value = ["probation-areas"],
+        produces = [APPLICATION_JSON_VALUE])
+@Slf4j
+class ProbationAreaController(val localDeliveryUnitService: LocalDeliveryUnit2Service) {
+
+    @GetMapping(
```

Comment:
If we do a UI for this we might need to retrieve all LDUs for probation areas

Reply:
Wouldn't you ask Delius what the Probation Areas, LDUs were?  This service doesn't have that information.

Reply:
Yes you would to get the relationship from delius. But you may want to decorate that info with info about mailboxes and to do that you'd want to bulk load of whatever infos available for a probation area might be good. 

(Still we'd only add it if we need it )